packages/cli/src/index.ts

```diff
@@ -81,31 +95,30 @@ async function main() {
     .option("-u, --url <url>", "RPC URL", "https://devnet.genesysgo.net")
     .option("-d, --dry-run", "Flag for dry run. Doesn't mint an NFT")
     .action(async ({ publisherMintAddress, keypair, url, dryRun }) => {
-      try {
+      tryWithErrorMessage(async () => {
         await checkForSelfUpdate();
 
         const config = await getConfigFile();
 
         if (!hasAddressInConfig(config.publisher) && !publisherMintAddress) {
-          showUserErrorMessage(
-            "Either specify an publisher mint address in the config file, or specify as a CLI argument to this command."
-          );
-          createCommand.showHelpAfterError();
-          return;
+          throw new Error("Either specify a publisher mint address in the config file or specify as a CLI argument to this command.")
         }
 
         const signer = parseKeypair(keypair);
         if (signer) {
-          await createAppCommand({
+          const result = await createAppCommand({
             publisherMintAddress: publisherMintAddress,
             signer,
             url,
             dryRun,
           });
+
+          const displayUrl = `https://solscan.io/token/${result.appAddress}${generateNetworkSuffix(url)}`;
+          const resultText = "App NFT successfully minted:\n" + displayUrl;
```

Comment:
consistent use of interpolation would be nice but not a blocker

Reply:
Can I fix after this PR? :)